domain-concept/src/main/java/life/qbic/domain/concepts/communication/EmailService.java

```diff
@@ -3,4 +3,6 @@
 public interface EmailService {
 
   void send(Email email);
+
+  void send(String recipient, String recipientFullName, String subject, String message);
```

Comment:
IMO this is unnecessary since we can just generate and provide the email record. (Even better if that's handled by the UserContactService similiar to the password reset email generation)

Reply:
The issue is, that Email is introduced as domain concept but with a technical low level detail: mime type. To not break existing functionality, I decided to override the method with simple String values.